.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+defaults: &defaults
+  working_directory: ~/javascript-utilities
+  docker:
+    - image: circleci/node:8.6.0-browsers
+
+version: 2
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-yarn-{{ checksum "yarn.lock" }}
+            - v1-yarn
+      - run: yarn
+      - save_cache:
+         key: v1-yarn-{{ checksum "yarn.lock" }}
+         paths:
+           - node_modules
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - javascript-utilities
+  
+  test:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+         at: ~/
+      # Circle cache magic:
+      # - Caches are immutable
+      # - If Circle cannot find an exact key match, it falls back to the closest partial match
+      # - If multiple partial matches are found, it falls back to the *most recent* partial match
+      #
+      # Appending an {{ epock }} to a build's cache allows rebuilds to find the most recent branch cache.
+      # For new branches, Circle falls back to matching the most recent master cache.
+      - restore_cache:
+          keys:
+            - v4-test-app-{{ .Branch }} # Match branch builds, favoring the most recent build.
+            - v4-test-app-master        # If there's no branch build match, use the most recent master cache.
+      - run: yarn run check
+      - save_cache:
+         key: v4-test-app-{{ .Branch }}-{{ epoch }}
```

Comment:
I think you don’t need to use -app in all these tasks

Reply:
fair enough. I just copied what we had in `web`.